src/modules/basic_auth/filter/basic_auth_filter.e

```diff
@@ -21,6 +21,8 @@ feature -- Basic operations
 		local
 			l_auth: HTTP_AUTHORIZATION
 		do
+				-- We can inherit from SHARED_LOGGER and write
```

Comment:
The SHARED_LOGGER should mainly be used before the CMS is really initialized.
After I would recommend using the CMS's logger object.
To be discussed